recipes/xspec-models-lite/startup.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
What is this file for?

Reply:
A leftover from building locally. It is removed now.